app/models/grda_warehouse/warehouse_reports/youth/homeless_youth_report.rb

```diff
@@ -589,7 +589,14 @@ def follow_up_two_c
     end
 
     def follow_up_two_d
-      @follow_up_two_d ||= follow_up_from_homelessness.pluck(:zip_code).uniq
+      @follow_up_two_d ||= begin
+        hmis_clients = GrdaWarehouse::HmisClient.where(client_id: follow_up_two_b) # clients w/ follow ups
```

Comment:
Isn't HmisClient.client_id a source_client id?

Reply:
You are right, It is. Good catch!